### PR TITLE
update aws-sdk-go/service/ebs

### PR DESCRIPTION
**Pull request by orouz**
see https://github.com/elastic/cloudbeat/pull/1799#issuecomment-1885806370

[pass 1](https://github.com/elastic/cloudbeat/actions/runs/7486844568/job/20378006647)
[pass 2](https://github.com/elastic/cloudbeat/actions/runs/7486844142/job/20379421972)

### CODE REVIEW

File: .github/workflows/cnvm-ci.yml
Comment (orouz) [inline .github/workflows/cnvm-ci.yml:1]: reverts https://github.com/elastic/cloudbeat/pull/1706

changed the timeout to 15 mins as all passing runs usually took about 10~ mins

we could add a `paths` filter but it's a bit unclear which files to include and we might miss some

**Review by oren-zohar — APPROVED** [review]

**Review by oren-zohar — COMMENTED** [review]
@orouz it used to be `pull-request` no `pull-request-target`, no?

**Comment by orouz** [comment]
> @orouz it used to be `pull-request` no `pull-request-target`, no?

it was `pull_request_target`, see https://github.com/elastic/cloudbeat/pull/1706/files

**Comment by orestisfl** [comment]
> > @orouz it used to be `pull-request` no `pull-request-target`, no?
> 
> it was `pull_request_target`, see https://github.com/elastic/cloudbeat/pull/1706/files

We want it to remain on post-merge though, right? The CI was flaky before it got completely broken.

**Pull request merged**